### DISCUSSION TITLE
Made some visual tweaks, what do you think?

### DISCUSSION THREAD

**Discussion by k4my4b** [the-djmaze/snappymail#970] [Ideas]
Hey, I decided to make some adjustments to snappymail, what do you think? worthwhile to push it upstream?
![nextcloud-snappy](https://user-images.githubusercontent.com/17426041/219106166-d8061b1d-a08d-406f-836e-6c57d459f4a2.png)

**Comment by cm-schl** [comment]
Hi,
could you point out a bit the adjustments you've made 🙂 ?

**Comment by k4my4b** [comment]
@cm-schl 

Hi there, here's a before and after (ignore nc menu bar, it was spasming out for some reason):
I came across snappymail a few days ago in my search for a functional mail client for Nextcloud, snappy seems to do the job 
but it looks a bit archaic and out of place so generally, I was wondering if this looks appealing to the general snappy audience 
or if everyone prefers the original aesthetics. (I'm unfamiliar with the project so I do not know if there's any active/ongoing work
to overhaul the UI, I can only assume it would be on the roadmap).  
![before](https://user-images.githubusercontent.com/17426041/219330534-2cb6e547-3462-46c4-bf60-0621183c76cd.png)
![after](https://user-images.githubusercontent.com/17426041/219330557-cc58e276-2493-4d41-b45a-07506e25c9b1.png)

**Comment by phsc84** [comment]
I like the look of your updated version, but the folder list should be more compact. I have quite a lot of folders and would have to scroll much more than today.

**Reply by k4my4b** [reply]
Like the original size/spacing? I would argue that doesn't look too good but where there's a will there's a way, plus there's a search function for folders as well so isn't that better than scrolling?

**Comment by the-djmaze** [comment]
I've made the message toolbar buttons visible on >1400px
